### PR TITLE
Default to Release build

### PR DESCRIPTION
**Pull request by bennibbelink**
Updates `install.py` to set `-DCMAKE_BUILD_TYPE=Release` if no `--build-type` is specified.  Removes this flag from the dockerfiles where we install cyclus.

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -79,6 +79,8 @@ def install_cyclus(args):
                           ]
         if args.build_type:
             cmake_cmd += ['-DCMAKE_BUILD_TYPE=' + args.build_type]
+        else:
+            cmake_cmd += ['-DCMAKE_BUILD_TYPE=Release']
```
Comment (gonuke) [inline install.py:83]: Or should we set a default in the argparse declaration of this CLI parameter?

**Review by gonuke — CHANGES_REQUESTED** [review]
If we set the default in argparse it will be visible to users

File: install.py (hunk 1 above)
Comment (bennibbelink) [reply install.py]: That is a better solution, thanks for the suggestion @gonuke

**Comment by gonuke** [comment]
Do we need the same change for Cycamore?

**Comment by bennibbelink** [comment]
Yes, please see https://github.com/cyclus/cycamore/pull/600

**Review by gonuke — APPROVED** [review]

**Pull request merged**